application_service/app_service.mak

```diff
@@ -52,9 +52,9 @@ all:	app_service.exe hello_world.exe send_request.exe test_user.exe
 
 clean:
 	@echo "removing object files"
-	rm $(O)/*.o
+	rm -rf $(O)/*.o
 	@echo "removing executable file"
-	rm $(EXE_DIR)/app_service.exe
+	rm -rf $(EXE_DIR)/app_service.exe
```

Comment:
As an example, without these changes, `make clean` would complain and fail with non-zero `$rc` as follows:

```
sdb-fdb-build:[32] $ make -f example_app.mak clean
removing object files
rm ./*.o
rm: cannot remove './*.o': No such file or directory
make: *** [example_app.mak:50: clean] Error 1

sdb-fdb-build:[33] $ echo $?
2
```

In future, if we ever build a CI-system, this non-zero `$rc` from `make clean` will needlessly cause automation scripts to fail.

So, this kind of rework is better and also improves usability by not printing noisy messages.